src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/ValueListBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can we rename the file to something like `ValueListBuilder.Pop.cs`?  The general convention we have with these partial files is that they're suffixed with the specialization they provide.